contracts/core/libraries/Metadata.sol

```diff
@@ -1,12 +1,10 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity 0.8.19;
 
-library Metadata {
-    struct MetaPtr {
-        /// @notice Protocol ID corresponding to a specific protocol.
-        /// More info at https://github.com/allo-protocol/contracts/tree/main/docs/MetaPtrProtocol.md
-        uint256 protocol;
-        /// @notice Pointer to fetch metadata for the specified protocol
-        string pointer;
-    }
-}
+struct Metadata {
```

Comment:
@codenamejason I removed the library to avoid having to write `Metadata.MetaPtr` everywhere. Lemme know if we want to change this back to an actual library